### ISSUE TITLE
How do I get an actual KML file of my Rectified map and not just a link?

### ISSUE THREAD

**Issue by wbbnm** [timwaters/mapwarper#196]
I am a new user to map_warper.  I recently Rectified a jpeg map.  The process worked well in map_warper.

I want to get the KML file so I can use the rectified map in other applications like Basecamp.

I exported the KML file, but apparently am only getting a link because the file size is 1 kb.  I was able to open the link and view my 
rectified map in Google Earth.  I tried saving the file as a kmz, but again only seemed to get a link since again the file size was 1kb

**Comment by timwaters** [comment]
Hi @wbbnm thanks for the issue. Its not currently available in mapwarper. The KML format can have many different things in it. The one you want is basically a wrapper around the entire image. You might be able to download the image, import into google earth and export it as KML. or there might be other software that would work.

**Comment by wbbnm** [comment]
Tim,

 

Thanks for the reply.  

 

I found a solution to my problem.  

 

After I looked closer at the Google Earth File\Open options, I saw I could open .tif files.  So I downloaded the one for my map and was able to open it in GE.  I was then able to create a kmz file which I then loaded into Basecamp.  I will use it as a Custom map overlay there.  This is what I really needed to do.

 

I was planning to add this info to my problem request,  but I see that it has been closed.

 

I think your software is going to be very useful in Rectifying map image files that don’t have lat/long info.

I have passed this along to some friends who will probably be using it as well.

 

Bill Boyer
